Mathlib/Algebra/Ring/Aut.lean

```diff
@@ -62,4 +62,29 @@ def toPerm : RingAut R →* Equiv.Perm R where
   map_one' := rfl
   map_mul' _ _ := rfl
 
+variable {R}
```

Comment:
Do you really need these declarations? I am a but surprised if they're really useful and we don't already have them.

Reply:
If the other structures already have those lemma it's OK, thanks!